bot.py

```diff
@@ -259,15 +259,15 @@ def registerTexts(self):
 
 /listall - Lists all possible groups to watch.
 
-/noplus1 - Enables +1 filtering.<b>Experimental</b>.
+/noplus1 - Enables +1 filtering.*Experimental*.
 
 /yesplus1 - Disables +1 filtering.
 
-/addalias STUDENT_NO - Adds given student number to aliases to receive mention notifications. STUDENT_NO must be in the form e?\d{6,7}.<b>Experimental</b>.
+/addalias STUDENT\_NO - Adds given student number to aliases to receive mention notifications. STUDENT\_NO must be in the form e?\d{6,7}.*Experimental*.
```

Comment:
could you move this to markdown commit ?

Reply:
Instead I moved the telegram mode change to this commit